### DISCUSSION TITLE
Cannot overwrite : my-account-offcanvas.php

### DISCUSSION THREAD

**Discussion by PGK37** [bootscore/bootscore#552] [Q&A] [answered]
Hi, i'm trying to overwrite this template : my-account-offcanvas.php

I've put my file in WooCommerce Directory in my ChildTheme but nothing happens…
These is my structure - The file is the same as Main theme
**Bootscore-child-main / woocommerce / myaccount / my-account-offcanvas.php**

When i modify the original in Main theme it works but not in the ChildTheme

Any idea of this problem ? Thank's for your help (and very good Frameworks ! Very nice job )

**Comment by crftwrk** [comment, marked as answer]
Yes, this is an own file and not a WooComnerce template. Check https://github.com/bootscore/bootscore/issues/61#issuecomment-1000734543 how to override this file in your child.

Solved?

**Reply by PGK37** [reply]
Merci Basti ! It work's perfect… sorry it was confusing for me ;-)

**Reply by crftwrk** [reply]
All good, almost nobody knows that 😉

Please mark the answer. Thank you.
